### PR TITLE
feat: add some logic

### PR DESCRIPTION
**Pull request by RashEstebek**

### CODE REVIEW

File: src/lib/components/Footer.svelte
Comment (Knm24) [inline src/lib/components/Footer.svelte:1]: someone needs to finish the design and layout

**Review by Knm24 — COMMENTED** [review]
good add style and css

File: src/lib/styles/header.css
[the remark this answers is not in the document]
Comment (Knm24) [reply src/lib/styles/header.css:1]: not done

File: src/lib/components/Header.svelte
[the remark this answers is not in the document]
Comment (RashEstebek) [reply src/lib/components/Header.svelte:1]: ok, we'll do it

File: src/lib/components/Footer.svelte
Comment (RashEstebek) [reply src/lib/components/Footer.svelte:1]: ok

File: src/routes/+layout.svelte
[the remark this answers is not in the document]
Comment (RashEstebek) [reply src/routes/+layout.svelte:1]: coool

**Comment by Knm24** [comment]
get planet id and submenu id for footer and header

**Review by rakhatovazarina — APPROVED** [review]

**Review by Knm24 — APPROVED** [review]

**Comment by Knm24** [comment]
merged

**Pull request merged**